src/index.tsx

```diff
@@ -25,15 +25,10 @@
 
 import * as React from 'react';
 import { Store } from 'redux';
-import { AAD_LOGIN_SUCCESS, AAD_LOGOUT_SUCCESS, loginSuccessful, logoutSuccessful } from './actions';
-import { IAuthProvider, IMsalAuthProviderConfig, IUserInfo, UserInfoCallback} from './Interfaces';
-import { MsalPopupAuthProvider } from './MsalPopupAuthProvider';
-import { MsalRedirectAuthProvider } from './MsalRedirectAuthProvider';
 
-enum LoginType {
-  Popup,
-  Redirect,
-}
+import { AAD_LOGIN_SUCCESS, AAD_LOGOUT_SUCCESS, loginSuccessful, logoutSuccessful } from './actions';
+import { IAuthProvider, IAuthProviderFactory, IUserInfo, LoginType, UserInfoCallback} from './Interfaces';
+import { MsalAuthProviderFactory } from './MsalAuthProviderFactory';
 
 enum AuthenticationState {
   Unauthenticated,
```

Comment:
we don't use Authenticating anymore right? should we remove any related logic to keep the code clean?

Reply:
We aren't using it, but we should be. There is issue #47 deals with the fallout of not using it currently